docs/vercel.json

```diff
@@ -1444,6 +1444,10 @@
       "source": "/en/latest/modules/chains/examples/graph_cypher_qa.html",
       "destination": "/docs/use_cases/graph/graph_cypher_qa"
     },
+    {
+      "source": "/en/latest/modules/chains/examples/graph_ontotext_graphdb_qa.html",
+      "destination": "/docs/use_cases/graph/graph_ontotext_graphdb_qa"
+    },
```

Comment:
only needed for when we've moved doc pages around and wanted to not break old links. no need to do this for new docs

Reply:
Tnx. Fixed